### ISSUE TITLE
Recommender: Inverted_index using idf does not store the first ID.

### ISSUE THREAD

**Issue by rimms** [jubatus/jubatus_core#209]
* config:

  ```
  {
    "converter" : {
      "string_rules":[
        {"key" : "*", "type" : "str", "sample_weight":"bin", "global_weight" : "idf"}
      ]
    },
    "method": "inverted_index"
  }
  ```

The steps are the following:

1. update_row("ID", {"key": "id"})
2. get_all_rows
    * expected: ["ID"]
    * actual: []

**Comment by kmaehashi** [comment]
This happens because global_weight is set to `idf`. Refs https://github.com/jubatus/jubatus/issues/638

**Comment by kazuki** [comment]
This approach ?
https://github.com/scikit-learn/scikit-learn/blob/51a765a/sklearn/feature_extraction/text.py#L991-L993

**Comment by kmaehashi** [comment]
@kazuki Thank you for the reference.

The way used by `scikit-learn` seems reasonable.  We may provide the "log+1" version of IDF with an alternate name like `idf1`.

**Comment by kmaehashi** [comment]
Fixed via #300.
